### ISSUE TITLE
Merge Zip files

### ISSUE THREAD

**Issue by jaisalj** [srikanth-lingala/zip4j#296]
Hi,
We have a use case where we have a large file of around 2GB. We split the file into smaller parts and transfer via an api.
The receiving api gets the request and zips this part. When the next parts are received, is it possible to add those parts to the existing zip.
Something like Zip1 + Zip2 + Zip3 + ..... = OriginalFile.zip ?

**Comment by srikanth-lingala** [comment]
Yes, you can merge split zip files with zip4j. [Here][1] is the documentation and a sample.

[1]: https://github.com/srikanth-lingala/zip4j#merging-split-zip-files-into-a-single-zip

**Comment by jaisalj** [comment]
The method mergeSplitFiles requires a split file which is created from split method. 
Suppose we want to merge to different zip files into one. Something like concatenation. Is this supported ?

**Comment by srikanth-lingala** [comment]
I understood your question wrong earlier. Unfortunately, such feature is not supported yet. It will be a good feature to add though. I will mark this thread as a new feature. But with the current tight schedule, I am not sure when I will be able to implement this. I will try to do this in the next days, but please don't count on it.

**Comment by jaisalj** [comment]
Thank you for the support Srikanth.

**Comment by srikanth-lingala** [comment]
Please leave it open. I will implement this feature in zip4j. I will close it when it is implemented.

**Comment by jaisalj** [comment]
To be clear I'll explain once again. Suppose we have a text file of 10mb, we split the content of this file in multiple parts.
The result can be achieved in two ways - 
1. Having a ZIP container and append the split files inside it without unpacking the zip everytime.
2. We have multiple Zip files (Zip1, Zip2 .....) We need to concatenate all the zip parts so that it results in creation of a Zip file that contains the original 10MB file inside it.

Let me know if you have any questions regarding this.
